src/systems/thruster/Thruster.hh

```diff
@@ -85,6 +85,8 @@ namespace systems
   ///                      [Optional, defaults to 1000N or 1000rad/s]
   /// - <min_thrust_cmd> - Minimum input thrust or angular velocity command.
   ///                      [Optional, defaults to -1000N or -1000rad/s]
+  /// - <deadband> - Deadband of the thruster. Absolute value below which the 
+  ///                thruster won't spin nor generate thrust
```

Comment:
Could you also mention that can be enabled/disabled via messages?

Reply:
sure !